src/Files.Backend/Services/Settings/IFoldersSettingsService.cs

```diff
@@ -53,6 +78,31 @@ public interface IFoldersSettingsService : IBaseSettingsService, INotifyProperty
 		/// </summary>
 		FolderLayoutModes DefaultLayoutMode { get; set; }
 
+		/// <summary>
+		/// Gets or sets a value indicating git status column's default width
+		/// </summary>
+		double GitStatusColumnWidth { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating git last commit date column's default width
+		/// </summary>
+		double GitLastCommitDateColumnWidth { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating git last commit message column's default width
+		/// </summary>
+		double GitLastCommitMessageColumnWidth { get; set; }
```

Comment:
```suggestion
		double GitCommitMessageColumnWidth { get; set; }
```

Reply:
I dont understand why you suggest that I should remove 'Last'.

Reply:
Should I create a new class named GitCommitItem for this? If we are going to add more git columns in the future, LastCommit words would be needed.

Reply:
It's your call but I think we should address that if/when we add more columns.

Reply:
For now, we should not remove any words from current variable names.